src/scripts/main.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('First promise was rejected');
+  }, 3000);
```

Comment:
The 'setTimeout' method will reject the promise after 3 seconds, regardless of whether the click event has occurred or not. This could lead to an unhandled promise rejection. It would be better to clear the timeout when the click event occurs to prevent the promise from being rejected after it has been resolved.